### ISSUE TITLE
TAURI_INVOKE_HANDLER in ubuntu 20.04

### ISSUE THREAD

**Issue by TheFern2** [tauri-apps/tauri#1069]
**Describe the bug**
TAURI_INVOKE_HANDLER on a react app with tauri. I built a project with cra cli utility, and then added tauri. Ran it and worked fine I could see GUI in both dev and build. I then wanted to test the IPC between js and rust, and calling the command kept getting the invoke handler error. I said ok maybe I am missing something, downloaded https://github.com/lukethacoder/tauri-react-example and I get the same error. So now I am lost.

**To Reproduce**
1. Create a cra app, add tauri
2. Call rust myCustomCommand cmd from App.js
3. See error

**Expected behavior**
Expected to see rust cmd print message on the console, and gui on dev server.

**Screenshots**
![image](https://user-images.githubusercontent.com/10265682/97632498-96f6a200-1a00-11eb-8d7c-0be5b150f877.png)

**Platform and Versions (please complete the following information):**
```
[tauri]: running info

Operating System - Ubuntu 20.04 Linux(5.4.0-51-generic) - linux/x64

Node.js environment
  Node.js - 12.18.3
  tauri.js - 0.12.0

Rust environment
  rustc - 1.47.0
  cargo - 1.47.0
  tauri-bundler - 0.9.4

Global packages
  NPM - 6.14.6
  yarn - Not installed

App directory structure
/.git
/build
/node_modules
/public
/src
/src-tauri

App
  tauri.rs - 0.9.2
  mode - embedded-server
  build-type - bundle
  CSP - default-src blob: data: filesystem: ws: http: https: 'unsafe-eval' 'unsafe-inline'
  distDir - ../build
  devPath - http://localhost:3000
```

**Comment by TheFern2** [comment]
I was running with react start script, rather than tauri dev. Mistakes were made. :disappointed:

**Comment by TheFern2** [comment]
Ok I still think is a bug, I think I should be able to start react.

**Comment by jbolda** [comment]
👋

Perhaps give the CLI a try? In the (slow progress but) process of improving it, but `mkdir test-tauri && cd test-tauri && yarn init && yarn add tauri && yarn tauri create`. This template also wires up the template to run on the start command.

**Comment by TheFern2** [comment]
:smiley:
https://tauri.studio/en/docs/api/cli These are the commands I used to  add tauri to my react project. Are you talking about another cli?

**Comment by TheFern2** [comment]
My code is identical to https://github.com/tauri-apps/examples/tree/dev/node/react/create-react-app I just added a bit of string to `src-tauri/main.rs` to say hello from rust. As far as I can tell I've created js functions, and used within react and it is all good. I can call Tauri.invoke to call myCustomCommand and that also works I can see the message printed on the terminal, but then react shows tauri_invoke_handler error on the browser which I guess is expected as rust can't run in the browser. Meanwhile tauri dev does shows react running fine in the dev app. So this isn't an issue just a bit of new stuff for me, brand new to rust and tauri.
